app/features/feedback/shared/strategies/single-answer-question/reducer.js

```diff
@@ -0,0 +1,11 @@
+import _ from 'lodash';
+
+function singleAnswerQuestionReducer(rule, annotations) {
+  const answerId = annotations || -1
+  const result = rule.answer === answerId.toString()
+  return _.assign({}, rule, {
```

Comment:
Can this use `Object.assign` instead of `_.assign`?

Reply:
sure